draft-ietf-ice-rfc5245bis.xml

```diff
@@ -4227,13 +4235,14 @@ spoofed ICMP attacks is possible also for off-path attackers from a node
 in a network without source address validation.
 </t>
 
-<t> Forcing the fake valid result works in a similar way. The agent
+<t>
+Forcing the fake valid result works in a similar way. The agent
 needs to wait for the Binding request from each agent, and inject a
-fake success response. The attacker won't need to worry about
-disrupting the actual response since, if the candidate is not valid,
-it presumably wouldn't be received anyway. However, like the fake
-invalid attack, this attack is mitigated by the STUN short-term
-credential mechanism in conjunction with a secure candidate exchange.
+fake success response, or route (e.g., using a tunnelling mechanism) a
+success response that normally would be dropped or rejected by the
+network to the agent. Again, due to the STUN short-term credential
+mechanism, in order for the attacker to inject a valid success response, the
+attacker needs the password.
```

Comment:
For the tunnelling attack no password is needed by the attacker (but this text seems to imply it's needed). To fix could move the last sentence before the "route attack" part and have the route attack part in the end.

Reply:
Will fix.